<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnFutureValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>FutureValue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCurrentValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>CurrentValue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultView">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>330</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Interest rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInterest">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>40</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAmount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>283</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_this_app"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_this_app">
   <property name="text">
    <string>About this app</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnFutureValue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calculateFutureValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCurrentValue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calculateCurrentValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculateFutureValue()</slot>
  <slot>calculateCurrentValue()</slot>
 </slots>
</ui>
